### DISCUSSION TITLE
Docs for the API?

### DISCUSSION THREAD

**Discussion by noconsulate** [resend/react-email#862] [Q&A]
I'm getting started with react-email and look forward to using it with resend. But I'm confused about the documentation. Is there anything for how to actually compose an email besides the extremely terse 'Getting Started' section and the examples? It seems like everything I do is generating errors and trial and, well, error is going to take more time than I wish!

**Comment by osdiab** [comment]
lol yeah, I'm finding between this library and `resend` having basically no docs and next to no configurability, that despite @zenorocha 's proclamations of how important dev experience is, the dev experience is pretty bad. Guess it comes with this being a project in its infancy.

**Reply by noconsulate** [reply]
Too bad they're active on their own discussion forum. Answering questions here would help them develop docs and perhaps more importantly make them seem like a serious project. Too bad because it seems like a great project and it tackles an unexploited niche! I can get by with resent/react-email for now but I'll be looking for something else probably. :(
